### PR TITLE
clean up ghc-api pragmas

### PR DESCRIPTION
**Pull request by pepeiborra**
We no longer depend on ghc-lib so it's cleanup time

### CODE REVIEW

**Comment by anka-213** [comment]
Oh, we don't? That'll make a huge difference for this PR https://github.com/haskell/haskell-language-server/pull/1649.

What was it used for before and what has changed?

**Comment by pepeiborra** [comment]
> Oh, we don't? Then I should probably also remove all the new references to it in this PR #1649.
> 
> What was it used for before and what has changed?

We dropped the dependency in https://github.com/haskell/haskell-language-server/pull/1366

**Comment by anka-213** [comment]
Ah, thanks!

**Review by ndmitchell — APPROVED** [review]
Great idea

File: plugins/hls-explicit-imports-plugin/hls-explicit-imports-plugin.cabal [hunk 1]
```diff
@@ -9,7 +9,6 @@ maintainer:         pepeiborra@gmail.com
 category:           Development
 build-type:         Simple
 extra-source-files:
-  include/ghc-api-version.h
```
Comment (Ailrun) [inline plugins/hls-explicit-imports-plugin/hls-explicit-imports-plugin.cabal:12]: It looks like actual file is not deleted?

**Pull request merged**